File: 215-kth-largest-element-in-an-array/kth-largest-element-in-an-array.cpp
class Solution {
public:
    int findKthLargest(vector<int>& nums, int k) {
        // I have to return the Kth largest element from the array 
        // By using minHeap I can solve 
        priority_queue<int,vector<int>,greater<int>>minHeap;

        for(int i=0;i<nums.size();i++){
            // I have return the kTH largest element bhaii 
            minHeap.push(nums[i]);

            if(minHeap.size()>k){
                minHeap.pop();
            }
        }
        return minHeap.top();
    }
};

File: 215-kth-largest-element-in-an-array/kth-largest-element-in-an-array.java
class Solution {
    public int findKthLargest(int[] nums, int k) {
        // minHeap 
        // PriorityQueue<Integer>minHeap = new PriorityQueue<>();

        // I can sort the array also,
        Arrays.sort(nums);
        int n = nums.length;

        return nums[n-k];
    }
}